pkgs/tools/text/tab/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromBitbucket, python2 }:
+
+stdenv.mkDerivation rec {
+  version = "7.1";
+  pname = "tab";
+  name = "${pname}-${version}";
+
+  src = fetchFromBitbucket {
+    owner = "tkatchev";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "049wx24b4v8alhvc05z7snyzaf6fk9w2npc4kklp366yjl1l4qbb";
+  };
+
+  buildInputs = [ python2 ];
```

Comment:
python2 is only used for running tests, so add this to nativeBuildInputs instead.